main.go

```diff
@@ -306,7 +306,10 @@ func main() {
 			log.Fatalf(err.Error())
 		}
 		dispatchTomb.Go(func() error {
-			go csLAPI.Run()
+			go func() {
+				csLAPI.Run()
+				log.Fatal("LAPI can't be reached")
```

Comment:
don't we want to check for error before fataling and/or provide the error to the user ?